tests/unit/config/test_configtypes_hypothesis.py

```diff
@@ -36,6 +36,8 @@ def gen_classes():
             pass
         elif member is configtypes.MappingType:
             pass
+        elif member in [configtypes.List, configtypes.LengthList]:
+            pass
```

Comment:
Yes, that works. I may not know how to Python but I know how to curry. I assume multiple yields to test different inner types is okay?